source/manual/business-readiness-publish-changes.html.md

```diff
@@ -46,9 +50,16 @@ The content item for the business readiness finder and its email signup page are
 
 9. If you are making changes to the schema, you may also need to [reindex elasticsearch](/manual/reindex-elasticsearch.html).
 
+## Updating the email signup page
+
+1. Update the [search-api email YAML file][search-api-email]
+2. Merge the changes
+3. Deploy search-api
+3. Run the [`publishing_api:publish_facet_group_eu_exit_business_finder` rake task][staging-rake-task-facet-group] in `search-api` to publish the changes
```

Comment:
Point `3.` is listed twice.

Also, can you add a note that rake task `publishing_api:publish_facet_group_eu_exit_business_finder` publishes both the finder and the email signup page at the same time.

Reply:
Good spot on the list numbering.

I'm not sure it's relevant to specify that the email signup page and the finder are both updated here - if the instructions are followed, the only visible change will be to the email signup page. There are already [instructions for running this rake task to publish the finder](https://github.com/alphagov/govuk-developer-docs/pull/1804/files#diff-41cb827b0c49ca27257f22e0d8a203d9R36) itself. Let me know if you'd still like me to add something.

Reply:
(If this is just a warning - in case the dev has updated the finder but didn't want to publish the changes yet - then I guess the same should be applied to [those instructions](https://github.com/alphagov/govuk-developer-docs/pull/1804/files#diff-41cb827b0c49ca27257f22e0d8a203d9R36) too? i.e. warn that publishing the finder will also publish the email signup page).

Reply:
I think that `N.B.` or similar should be added somewhere so people aren't surprised by any unexpected changes.

Reply:
I think it's ok to only add it once as it isn't a very long page.